lib/tech_for_good_near_you/meet_ups/lat_lon.ex

```diff
@@ -0,0 +1,21 @@
+defmodule TechForGoodNearYou.LatLon do
```

Comment:
I think this should live in the web folder, it's not really anything to do with our models, maybe it could go in a folder called requests?